turbo.json

```diff
@@ -1,39 +1,54 @@
 {
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": ["package.json"],
   "pipeline": {
     "build": {
-      "dependsOn": ["^build", "intl-compile", "codegen"],
+      "dependsOn": ["^build", "build:css", "intl-compile", "codegen"],
       "inputs": ["src/**", "package.json"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**"],
+      "outputMode": "new-only"
     },
     "build:css": {
-        "outputs": ["src/assets/css/hydrogen.css"]
+      "inputs": ["src/**"],
+      "outputs": ["src/assets/css/hydrogen.css"]
     },
     "codegen": {
       "inputs": ["src/**/*.graphql", "../../packages/**/src/**/*.graphql"],
-      "outputs": ["src/api/generated.ts", "src/index.ts"]
+      "outputs": ["src/api/generated.ts", "src/index.ts"],
+      "outputMode": "new-only"
     },
     "dev": {
       "dependsOn": ["^build", "build:css", "intl-compile", "codegen"],
       "inputs": ["src/**", "package.json"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**"],
+      "outputMode": "new-only"
     },
     "watch": {
       "dependsOn": ["build:css", "intl-compile", "codegen"],
-      "cache": false
+      "cache": false,
+      "persistent": true
     },
     "build:intl-cli": {
-      "outputs": ["dist/cli.js"]
+      "inputs": ["src/cli.ts"],
+      "outputs": ["dist/cli.js"],
+      "outputMode": "new-only"
     },
     "intl-extract": {
-      "outputs": ["src/lang/en.json"]
+      "inputs": ["src/**"],
```

Comment:
Worth narrowing to just ts and tsx?

Reply:
Yeah, I think I missed the mark on all these scripts inputs. Updated in [update inputs to make it more specific](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5909/commits/f0fe86dd9c7b0dd8711cd161a2668717d6cdb8a2)